torch/_export/utils.py

```diff
@@ -860,6 +860,10 @@ def _extract_pytree_key(x):
         if node.op == "placeholder":
             assert node.name in name_map
             node.name = node.target = name_map[node.name]
+            # if the constant obj is an input, we also need to update meta["val"]
```

Comment:
Can we leave this place unchanged and somehow mutates the name map in the for loop in L833?

Reply:
Sure